### DISCUSSION TITLE
Community calls and questions

### DISCUSSION THREAD

**Discussion by MEK3DK** [microsoft/microsoft-ui-xaml#8927] [General]
I know that there will not be any more community calls like that one that just happened, because the format is moving to new name and quarterly time schedule. However you ask people to provide questions and there a lots of them in #8884. Then at the meeting it was VERY clear that nobody from the team had looked through the questions and the meeting was cut short because there was no questions...
Please assign somebody (this has never worked perfectly) to gather up ALL questions, you don't have to answer them if you don't want to/don't like them, or whatever, but please at least know that they are there.

**Comment by knightmeister** [comment]
I posted this in #8884 but duplicating here because relevant.

How insanely out of touch are you guys with the general feeling from your involved developers (i.e. us)?
This call was a complete and utter waste of time.

What is the point of announcing the Community Call and soliciting feedback for it to be entirely ignored?

I'm amazed that you guys ignore the top rated posts ITT. Clearly there is a large amount of dissatisfaction with the progress and management of WinUI - please affect positive change and address the numerous issues around management, communication, backlog, etc.

What is the point of having issues enabled here if you're not going to use them? The backlog on Github is just going to get back to the state that it was. Can we at least get bi-directional updates in place so that when the team is looking at an issue in ADO we can see progress on GH?

And I'm still waiting to hear how many people are actively working on this product....

**Comment by duncanmacmichael** [comment]
Already ahead of you! I just posted a follow-up to the community call [here](https://github.com/microsoft/microsoft-ui-xaml/discussions/8980) with answers to all the questions that we didn't have time to answer during the call, and in fact was working on gathering up answers when you posted this. :) Thanks for your diligence!
